stix_shifter_modules/guardium/configuration/config.json

```diff
@@ -23,24 +42,36 @@
             "type": "password",
             "optional": true
         },
-        "client_id": {
-            "type": "text",
-            "previous": "connection.options.client_id"
-        },
-        "client_secret": {
-            "type": "password",
-            "previous": "connection.options.client_secret"
-        }
+        "options": {
```

Comment:
I already mentioned in the previous pr. not sure why `client_id`,  `client_secret` and `grant_type` put inside `options`.  it can stay inside top level of connection object

Reply:
Removed.